packages/notification-system/src/components/NotificationsDashboard/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import NotificationCard from '../NotificationCard';
+import styles from './NotificationsDashboard.module.scss';
+
+const NotificationsDashboard = () => {
+  return (
+    <>
+      <div className={styles.dashboard}>
```

Comment:
Is this the view component? If yes som recomendations:

1. This is not a componente per se, this is a view, so what if we name the file `NotificationsDashobard.view` and we put it inside a `views` dir?
2. Lets simulate the request using a `useEffect` and a `setTimeout`, take look of `useState` hook too

Reply:
1. component -> view. ✔

Reply:
Oh I mean, that we can add suffixes indicating what a file contains, for example, for redux, we can have:

1. `something.slice.tsx`
2. `something.actions.tsx`
3. `something.reducers.tsx`

Is this clear?

Reply:
Oh, great!. I got it now. Ty 🙇🏻‍♂️